recipes-nodes/rbuilder/rbuilder_dev.bb

```diff
@@ -1,21 +1,14 @@
 SUMMARY = "Rust Builder"
-HOMEPAGE = "https://github.com/flashbots/rbuilder-private"
+HOMEPAGE = "https://github.com/flashbots/rbuilder"
 LICENSE = "CLOSED"
 
 include rbuilder.inc
 
 python () {
     import os
     origenv = d.getVar("BB_ORIGENV", False)
-    git_token = origenv.getVar('GIT_TOKEN') or os.getenv('GIT_TOKEN')
-    if not git_token:
-        bb.note("GIT_TOKEN environment variable not set. Will set SRC_URI with ssh protocol")
-        # Set the SRC_URI with the ssh protocol
-        d.setVar('SRC_URI', f"git://git@github.com/flashbots/rbuilder-private;protocol=ssh;branch=main")
-    else:
-        bb.note("GIT_TOKEN is set")
-        # Set the SRC_URI with the token included
-        d.setVar('SRC_URI', f"git://{git_token}@github.com/flashbots/rbuilder-private.git;protocol=https;branch=main")
+    d.setVar('SRC_URI', f"https://github.com/flashbots/rbuilder;branch=main")
+    d.setVar('SRC_URI[sha256sum]', f"ba8befe8c83bf7340373ce132d9c8ccac308fd7551ac73849a4c6937de848e12")
```

Comment:
are you choosing here a specific commit?
If yes, then you can put that down in the SRCREV = "ba8befe8c83bf7340373ce132d9c8ccac308fd7551ac73849a4c6937de848e12"

Also you can simplify all of this by removing the whole python block and have 

SRC_URI = "https://github.com/flashbots/rbuilder;branch=main"
SRCREV = "ba8befe8c83bf7340373ce132d9c8ccac308fd7551ac73849a4c6937de848e12"

I did the python stuff because it was private repo and we needed a way to inject either the git token or use ssh to allow yocto pulling and building it